CHANGELOG.md

```diff
@@ -587,23 +587,23 @@ From PWA 6.0.0, we support semantic versioning.
 
 ### Code Refactoring
 
-- cleanup/remove obsolete/update TODO comments ([e43c432](https://github.com/intershop/intershop-pwa/commit/e43c432))
-- replace deprecated formly field configuration properties ([fad6075](https://github.com/intershop/intershop-pwa/commit/fad6075))
+- cleanup, remove obsolete, and update TODO comments ([e43c432](https://github.com/intershop/intershop-pwa/commit/e43c432))
+- replace deprecated Formly field configuration properties ([fad6075](https://github.com/intershop/intershop-pwa/commit/fad6075))
 - **Angular 15:** replace class-based route guards by functional guards ([289e161](https://github.com/intershop/intershop-pwa/commit/289e161))
 - **Angular 15:** rework translate loader on SSR after Angular 15 update ([292d559](https://github.com/intershop/intershop-pwa/commit/292d559))
 
 ### BREAKING CHANGES
 
 - The `TODO` code cleanup removed some implementation fixes to work with ICM versions prior to 7.10.39.2.
-- **Angular 15:** Class-based route guards are replaced by functional guards (see [Migrations / From 3.3 to 4.0](https://github.com/intershop/intershop-pwa/blob/develop/docs/guides/migrations.md#from-33-to-40) for more details).
+- **Angular 15:** Class-based route guards were replaced by functional guards (see [Migrations / From 3.3 to 4.0](https://github.com/intershop/intershop-pwa/blob/develop/docs/guides/migrations.md#from-33-to-40) for more details).
 - **Angular 15:** ICM Translate Loader needs new configuration for `LOCAL_TRANSLATIONS` (see [Migrations / From 3.3 to 4.0](https://github.com/intershop/intershop-pwa/blob/develop/docs/guides/migrations.md#from-33-to-40) for more details).
-- **Angular 15:** Angular 15 and dependencies update (removed Browserslist configuration, update TypeScript compiler `target`, Stylelint adaptions, Jest tests adaptions).
+- **Angular 15:** Angular 15 and dependencies update (removed Browserslist configuration, update TypeScript compiler `target`, Stylelint adaptations, Jest tests adaptations).
 - Intershop PWA development is now based on Node.js 18.15.0 with npm 9.5.0 and the `"lockfileVersion": 3,`.
```

Comment:
Is the comma correct in `"lockfileVersion": 3,` ?

Reply:
This is okay and a copy from the actual configuration https://github.com/intershop/intershop-pwa/blob/develop/package-lock.json#L4
Besides this we will not start to edit old generated content in the change log.